package.json

```diff
@@ -21,6 +21,7 @@
     "prepare": "husky install",
     "thv": "ts-node ./utils/fetch-thv.ts --run",
     "test": "vitest",
+    "test:nonInteractive": "vitest run",
```

Comment:
I just added this so that a cursorrule/similar can easily be created to always run tests before and after making any changes.